test/models/concerns/audit_extensions_test.rb

```diff
@@ -35,4 +35,10 @@ def setup
     a = Audit.where(auditable_type: 'Setting')
     assert_equal "[encrypted]", a.last.audited_changes["value"][1]
   end
+
+  test "search for type=lookupvalue in audit" do
+    key   = VariableLookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first)
+    LookupValue.create!(:value => "ntp.mydomain.net", :match => "domain =  mydomain.net", :lookup_key => key)
```

Comment:
please use [existing factories](https://github.com/theforeman/foreman/blob/develop/test/factories/puppet_related.rb#L66) instead, here and on line above